crates/sim2h/src/lib.rs

```diff
@@ -801,12 +801,13 @@ fn spawn_handle_message_publish_entry(
             if let Some(uri) = state.lookup_joined(&space_hash, &agent_id) {
                 sim2h_handle.send((&*agent_id).clone(), uri.clone(), &multi_message);
             }
+            /* send not guaranteed to work so we can't mark as held.
             sim2h_handle.state().spawn_agent_holds_aspects(
                 (&*space_hash).clone(),
                 (&*agent_id).clone(),
                 data.entry.entry_address.clone(),
                 aspect_list.clone(),
-            );
+            );*/
```

Comment:
Yes, I regard this as the real fix for handling the case of failed holding requests.